doc/install/fast-install-and-first-run.rst

```diff
@@ -74,6 +91,7 @@ Setup
    # echo '</Location>' >> conf-enabled/ivre.conf
    # sed -i 's/^\(\s*\)#Rewrite/\1Rewrite/' /etc/dokuwiki/apache.conf
    # echo 'WEB_GET_NOTEPAD_PAGES = "localdokuwiki"' >> /etc/ivre.conf
+   # chmod o+r /usr/local/share/ivre/web/wsgi/app.wsgi
```

Comment:
When & why is that needed?

Reply:
Apache Web server is not allowed to read this file as www-data. I admit `chmod o+r` is not a clean way.
```suggestion
   # chgrp www-data /usr/local/share/ivre/web/wsgi/app.wsgi
```

Reply:
No I think it is OK to `chmod +r` for such a file, but I don't understand how it could end up without those rights, since this file should be world readable.

Reply:
Without this chmod, we see wsgi errors in Apache's error.log.